manage/deploying/wsgi.rst

```diff
@@ -17,8 +17,8 @@ Plone and WSGI
 
 Collection of Links about this topic, since this is a very special topic we also link to documentation about WSGI and Plone 4 for reference.
 
-* `Zope 4 and WSGI <https://zope.readthedocs.io/en/latest/wsgi.html>`_
+* `Zope 4 and WSGI <https://zope.readthedocs.io/en/latest/operation.html#using-alternative-wsgi-server-software>`_
 
-* `Plone 4 with WSGI <http://comments.gmane.org/gmane.comp.web.zope.plone.devel/23886>`_
+* `Plone 4 with WSGI <https://comments.gmane.org/gmane.comp.web.zope.plone.devel/23886>`_
```

Comment:
This link never connects. It appears that gmane.org has been out of commission for about a year. So I dug around in the Wayback Machine to find this link.

https://web.archive.org/web/20100702205219/http://permalink.gmane.org:80/gmane.comp.web.zope.plone.devel/23897

Is that relevant? It's before my time.